electricitylci/process_dictionary_writer.py

```diff
@@ -217,6 +226,50 @@ def exchange_table_creation_input_genmix(database, fuelname):
     return ar
 
 
+def exchange_table_creation_input_usaverage(database, fuelname):
+    """Add docstring."""

```

Comment:
would be nice to get this done before merging.

Reply:
Hi @jump2conclusionsmatt  I did not understand this part. Are you talking about the """docstring"""? I am not sure about that. @hottleta  Can you please help with the docstring?

Reply:
@jump2conclusionsmatt I would need some explanation as well

Reply:
Yup, I'm just talking about filling out this docstring, as well as the others.